polyval/src/field/backend/soft.rs

```diff
@@ -0,0 +1,72 @@
+//! Software emulation support for CLMUL hardware intrinsics.
+//!
+//! WARNING: Not constant time! Should eventually be disabled-by-default.
```

Comment:
TODO: make this constant time or disable it (by default, at the very least)

Reply:
Disabled by default and gated under the `insecure-soft` flag.

TODO: switch from a naive implementation to a "performance"-oriented (~150MB/sec) constant time fallback, ala Thomas Pornin's implementation (of GHASH, not POLYVAL, but same idea) in BearSSL:

https://bearssl.org/gitweb/?p=BearSSL;a=blob;f=src/hash/ghash_ctmul64.c